src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -79,14 +90,15 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
   const [DUC, setDUC] = useState<DataAccessDoc>()
   const [IRB, setIRB] = useState<DataAccessDoc>()
   const [attachments, setAttachments] = useState<DataAccessDoc[]>([])
-  const [accessorProfiles, setAccessorProfiles] = useState<UserProfile[]>([])
   const [
     formSubmitRequestObject,
     setFormSubmitRequestObject,
-  ] = useState<RequestInterface|RenewalInterface>()
+  ] = useState<RequestInterface | RenewalInterface>()
   const [alert, setAlert] = useState<AlertProps | undefined>()
   const [isRenewal, setIsRenewal] = useState<boolean>(false)
-  const requestedFileTypes = {}
+  const [accessors, setAccessors] = useState<Accessor[]>([])
```

Comment:
Update the state variable to not just contain a list of user profiles (for display usernames) but also the user's access type (for display radio buttons If `isRenewal === true`)